main.py

```diff
@@ -1,6 +1,12 @@
 """ToDo."""
 from tasks.kyu8 import example
+from tasks.kyu7 import where_is_vasya
 
+def find_vasya():
+    """Find a number of possible positions Vasya can ocupy."""
+    total = input("number of people in line: ")
```

Comment:
Add check (try...except) and rewrite all input like length = float(input("length: ")). Also rewrite (if '__name__'=='__main__').